flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/physical/stream/StreamPhysicalMultiJoin.java

```diff
@@ -263,7 +264,31 @@ public boolean inputUniqueKeyContainsCommonJoinKey(int inputId) {
         }
 
         final ImmutableBitSet commonJoinKeys = ImmutableBitSet.of(commonJoinKeyIndices);
-
         return inputUniqueKeys.stream().anyMatch(uniqueKey -> uniqueKey.contains(commonJoinKeys));
     }
+
+    private List<InputProperty> createInputProperties() {
+        final List<InputProperty> inputProperties = new ArrayList<>();
+
+        for (int i = 0; i < inputs.size(); i++) {
```

Comment:
nit: `for (int i = 0; i < inputs.size(); i++) { `-> `for (InputProperty inputProperty: inputs) {`

Reply:
I need the input's index here, take a look at the next line